packages/cli/src/util/get-pagination-opts.ts

```diff
@@ -14,8 +14,10 @@ export function getPaginationOpts(opts: PaginationOptions) {
     typeof limit === 'number' &&
     (!Number.isInteger(limit) || limit > 100 || limit < 1)
   ) {
-    throw new Error('Please provide an integer from 1 to 100 for option --limit');
+    throw new Error(
+      'Please provide an integer from 1 to 100 for option --limit'
+    );
   }
 
-  return [nextTimestamp, limit];
+  return [nextTimestamp, limit || 20];
```

Comment:
Lets change this to an object so it can be referenced by property name

```suggestion
  return { nextTimestamp, limit: limit || 20 };
```

Reply:
+1

Reply:
Rightio, changed that, but then changed the input to `getAliases` to an object. This change means changing a few more functions and all the calls to them, so thought I'd get a quick 👍🏾  at that before changing a bunch more stuff. 

https://github.com/vercel/vercel/pull/9102/commits/361d261be0f8f89f55ac71477ed809ae94685515

Reply:
So far so good!